radxa/dragon-q6a/default.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+{
+  imports = [
+    ../.
+  ];
+
+  config = {
+    hardware = {
+      radxa.enable = true;
+    };
+
+    boot = {
+      # We need a out-of-tree kernel for Dragon Q6A, otherwise important hardware such as NVME won't work.
+      kernelPackages = pkgs.linuxPackagesFor (import ./kernel.nix { inherit lib pkgs; });
```

Comment:
It will be part of upstream 6.19. I hope we can wait and just not deal with downstream kernel.

At least this should be `lib.mkDefault`.